--- stock
+++ first-A05
@@ -54,7 +54,7 @@
 G1 X85.796 Y88 E.81237
 G1 X86.187 Y86.747 E.04443
 G1 X86.374 Y86.601 E.00804
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X91.796 Y91.973 Z.6 F7200
@@ -242,7 +242,7 @@
 G1 X88.98 Y109.969 E.05093
 G1 X88.98 Y110.552 E.01999
 G1 X89.631 Y111.203 E.03154
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S214
 ;LAYER_CHANGE
 ;Z:0.4
@@ -250,7 +250,7 @@
 ;_SET_FAN_SPEED_CHANGING_LAYER
 SET_VELOCITY_LIMIT ACCEL=500 ACCEL_TO_DECEL=250
 M140 S35 ; set bed temperature
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X97.263 Y111.246 Z.8 F7200
@@ -436,12 +436,12 @@
 G1 X89.021 Y90.629 E.07696
 G1 X89.021 Y90.053 E.01949
 G1 X90.236 Y88.838 E.05818
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:0.6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X95.457 Y94.405 Z1 F7200
@@ -633,13 +633,13 @@
 G1 X89.021 Y109.371 E.07696
 G1 X89.021 Y109.947 E.01949
 G1 X90.236 Y111.162 E.05818
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S250
 ;LAYER_CHANGE
 ;Z:0.8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X97.869 Y111.221 Z1.2 F7200
@@ -699,12 +699,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:1
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z1.4 F7200
@@ -765,12 +765,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:1.2
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z1.6 F7200
@@ -829,12 +829,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:1.4
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z1.8 F7200
@@ -895,12 +895,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:1.6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z2 F7200
@@ -960,12 +960,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:1.8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z2.2 F7200
@@ -1025,12 +1025,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:2
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z2.4 F7200
@@ -1090,12 +1090,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:2.2
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z2.6 F7200
@@ -1155,12 +1155,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:2.4
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z2.8 F7200
@@ -1220,12 +1220,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:2.6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z3 F7200
@@ -1287,12 +1287,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:2.8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z3.2 F7200
@@ -1351,12 +1351,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:3
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z3.4 F7200
@@ -1417,12 +1417,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:3.2
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z3.6 F7200
@@ -1481,12 +1481,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:3.4
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z3.8 F7200
@@ -1547,12 +1547,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:3.6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z4 F7200
@@ -1613,12 +1613,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:3.8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z4.2 F7200
@@ -1678,12 +1678,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:4
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z4.4 F7200
@@ -1743,12 +1743,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:4.2
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z4.6 F7200
@@ -1808,12 +1808,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:4.4
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z4.8 F7200
@@ -1873,12 +1873,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:4.6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z5 F7200
@@ -1939,12 +1939,12 @@
 G1 X90.83 Y110.979 E.09743
 M73 P47 R6
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:4.8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z5.2 F7200
@@ -2004,12 +2004,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:5
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z5.4 F7200
@@ -2070,12 +2070,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:5.2
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z5.6 F7200
@@ -2134,12 +2134,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:5.4
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z5.8 F7200
@@ -2200,12 +2200,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:5.6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z6 F7200
@@ -2264,12 +2264,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:5.8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z6.2 F7200
@@ -2331,12 +2331,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z6.4 F7200
@@ -2396,12 +2396,12 @@
 G1 X89.021 Y93.708 E.22437
 G1 X89.021 Y90.83 E.09743
 G1 X91.013 Y88.838 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:6.2
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.129 Y94.502 Z6.6 F7200
@@ -2461,12 +2461,12 @@
 G1 X93.708 Y110.979 E.22437
 G1 X90.83 Y110.979 E.09743
 G1 X88.838 Y108.987 E.09534
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 ;LAYER_CHANGE
 ;Z:6.4
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X96.429 Y109.776 Z6.8 F7200
@@ -2535,13 +2535,13 @@
 G1 X89.428 Y93.301 E.18539
 G1 X89.428 Y90.422 E.09743
 G1 X90.606 Y89.245 E.05637
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S235
 ;LAYER_CHANGE
 ;Z:6.6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X95.828 Y94.811 Z7 F7200
@@ -2732,13 +2732,13 @@
 G1 X109.725 Y89.042 E.09103
 G1 X110.362 Y89.042 E.03325
 G1 X111.16 Y89.841 E.05898
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S206
 ;LAYER_CHANGE
 ;Z:6.8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X111.219 Y97.473 Z7.2 F7200
@@ -3196,13 +3196,13 @@
 G1 X110.511 Y89.489 E.05639
 G1 X108.845 Y89.489 E.05639
 G1 X110.483 Y91.127 E.0784
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S191
 ;LAYER_CHANGE
 ;Z:7
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X110.574 Y98.759 Z7.4 F7200
@@ -4046,13 +4046,13 @@
 G1 X89.477 Y91.602 E.00697
 G1 X89.341 Y91.664 E.00453
 G1 X88.835 Y91.159 E.02171
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S194
 ;LAYER_CHANGE
 ;Z:7.2
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X94.123 Y96.663 Z7.6 F7200
@@ -4914,13 +4914,13 @@
 G1 X108.388 Y96.233 E.01095
 ;WIDTH:0.357824
 G1 X108.294 Y96.373 E.00442
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S209
 ;LAYER_CHANGE
 ;Z:7.4
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X103.286 Y102.133 Z7.8 F7200
@@ -5566,13 +5566,13 @@
 G1 X92.248 Y109.085 E.02141
 G1 X91.724 Y108.608 E.02132
 G1 X90.189 Y107.073 E.06529
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S229
 ;LAYER_CHANGE
 ;Z:7.6
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 M73 P91 R0
@@ -6118,13 +6118,13 @@
 G1 X94.736 Y95.688 E.01991
 G1 X95.154 Y95.226 E.02137
 G1 X96.31 Y94.07 E.05605
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S245
 ;LAYER_CHANGE
 ;Z:7.8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X103.912 Y93.39 Z8.2 F7200
@@ -6540,13 +6540,13 @@
 G1 X97.469 Y101.981 E.01607
 G1 X97.556 Y102.083 E.00475
 G1 X98.526 Y103.053 E.0487
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 M106 S252
 ;LAYER_CHANGE
 ;Z:8
 ;HEIGHT:0.2
 ;_SET_FAN_SPEED_CHANGING_LAYER
-; printing object shallow_sphere.stl id:0 copy 0
+; printing object shallow_sphere.stl id:72340172838076673 copy 0
 SET_VELOCITY_LIMIT ACCEL=1000 ACCEL_TO_DECEL=500
 G1 E-.8 F1800
 G1 X103.103 Y102.85 Z8.4 F7200
@@ -6730,7 +6730,7 @@
 G1 X97.578 Y96.902 E.01773
 G1 X97.34 Y97.103 E.00924
 G1 X96.408 Y98.035 E.03914
-; stop printing object shallow_sphere.stl id:0 copy 0
+; stop printing object shallow_sphere.stl id:72340172838076673 copy 0
 G1 E-.8 F1800
 M106 S0
 ;TYPE:Custom
